### PR TITLE
disable firewalld for pulp and pulp docker to avoid push failure with…

### PR DESCRIPTION
**Pull request by zhangjing1**
… no route to host error

### CODE REVIEW

File: auto_testing_CI/initial_pub_pulp_psi.sh [hunk 1]
```diff
@@ -260,5 +265,10 @@ get_all_product_versions_content
 check_and_initialize_pub
 check_and_initialize_pulp_rpm
 check_and_initialize_pulp_docker
+for server in ${pulp_rpm_server} ${pulp_docker_server} 'docker-e2e.usersys.redhat.com'
+do
```
Comment (testcara) [inline auto_testing_CI/initial_pub_pulp_psi.sh]: what is the 'docker-e2e' used for?

File: auto_testing_CI/initial_pub_pulp_psi.sh [hunk 2]
```diff
@@ -249,6 +249,11 @@ clean_running_and_free_pub_tasks() {
     fi
 }
 
+disable_firewall_service(){
+  echo "== Disable the firewalld service on ${1}"
+  ssh -o "StrictHostKeyChecking no" root@${1} 'if [[ $(systemctl is-active firewalld) =~ "active" ]]; then systemctl stop firewalld; fi'
```
Comment (testcara) [inline auto_testing_CI/initial_pub_pulp_psi.sh]: We need to use the following  format:
ssh -i /root/.ssh/id_rsa -o "StrictHostKeyChecking no"

PSI is using the random user to run pod and script, we need to specify the password file to establish the ssh connection.

Comment (testcara) [reply auto_testing_CI/initial_pub_pulp_psi.sh]: and what is the 'docker-e2e' used for?

Comment (zhangjing1) [reply auto_testing_CI/initial_pub_pulp_psi.sh]: Thanks Cara. "docker-e2e" is the Docker Deamon which is used as a docker service for docker pull/push from brew pulp.
The firewalld on docker-e2e can also bring the push failure as here:
https://docs.engineering.redhat.com/display/PDT/E2E+Testing+Debug+Tips#E2ETestingDebugTips-9."Noroutetohost"onpulp-docker-e2eanddocker-e2e

**Pull request closed without merging**